### ISSUE TITLE
Accessibility: The listbox container's accessible name for search results is too verbose for screen readers.

### ISSUE THREAD

**Issue by MarcoZehe** [microsoft/vscode#93516]
- VSCode Version: 1.44 insiders
- OS Version: Windows 10 (18363)

Steps to Reproduce:

1. With NVDA on Windows, open any search or suggestion autocomplete situation. CTRL+P, CTRL+Shift+O, etc.
2. Type in something.
3. DownArrow into the search results.

* Expected result: NVDA should speak something short like "Suggestions listbox", followed by the first item that is selected.
* Actual result: NVDA speaks a long string of text, sometimes if you let it run and have many search results, this can take minutes, followed by "listbox", followed by the first selected item.

The reason is that the container, the item with role = "listbox", gets the accessible name from all the children. The string gets concatenated from all the search result items contained within. This is undesired. Instead, the item with role="listbox" should get an aria-label with a localized short phrase like "Suggestions" or "Results", so that the first item gets spoken instantaneously after entering the container.

Does this issue occur when all extensions are disabled?: Yes

**Comment by MarcoZehe** [comment]
CC @isidorn

**Comment by MarcoZehe** [comment]
Updated initial description with correct role, should have been "listbox" instead of "list" of course.

**Comment by MarcoZehe** [comment]
It appears that there is no aria-label set on the list, so Chromium tries to calculate it from the sub tree. Would it be enough to set the ariaLabel property [here](https://github.com/microsoft/vscode/blob/master/src/vs/editor/contrib/suggest/suggestWidget.ts#L607) to a localized string like 'Suggestions'?

**Comment by isidorn** [comment]
@MarcoZehe thanks for filling this issue.
Let's first figure something out. Are you using the suggest widget, or the quick pick widget.
From your keybindings Ctrl + P, Ctrl + SHift + O that is the quick pick widget and I changed its' role to be listbox today. Captured in this issue, so this might be a duplicate https://github.com/microsoft/vscode/issues/93464

If you are ussing a suggest widget that is trigger via ctrl+space in the editor and for that your code pointer holds true.

So let's first figure out if it is a duplicate (in which case it is, it will be fixed on Monday).
Let me know what you think, thanks!

**Comment by MarcoZehe** [comment]
Oh, then it‘s the quick picker. But I thought I heard something similar when typing in code and an auto suggestion came up. But I‘ll recheck both on Monday.

**Comment by isidorn** [comment]
Good, then I believe it is a duplicate which @neurrone already submitted.
Yeah the best is to recheck on Monday and let me know if you still see weirdness. Thanks!

**Comment by MarcoZehe** [comment]
This now works as expected in the Monday insider snapshot. It is indeed a dupe of #93464. Closing.

**Comment by isidorn** [comment]
Great, thanks for letting us know.
